app.rb

```diff
@@ -135,6 +107,73 @@
   redirect to("/merchant/#{merchant.public_id}")
 end
 
+def _user_and_business(country_code)
```

Comment:
Small nit and this is just my preference, but a lot of the information between USA and GBR are identical. We can probably declare the hash with the shared key values and then just merge in the attributes we care about in the `case`.